package.json

```diff
@@ -91,13 +97,14 @@
     "start": "nf start -p 3000 && electron-start",
     "electron-start": "ELECTRON_IS_DEV=1 node public/start-react",
     "electron": "yarn build-electron && electron .",
-    "react-start": "BROWSER=none react-scripts start",
+    "react-start": "react-scripts start",
```

Comment:
Does this change do anything?

Reply:
I shouln't commit this. On windows it gives an error, I think we will need a separate react-start for windows.